File: prime.py
import math

def isPrime(n):
	sn = math.floor(math.sqrt(n))
	for i in range(2,sn+1):
		if n % i == 0:
			return False
	return True

print("Primes.")
c = 0
for i in range(2,1_000_001):
	if isPrime(i):
		c += 1
print("Done.",c)

File: prime.js
function isPrime(n){
	let sn = Math.floor(Math.sqrt(n));
	for(let i = 2;i <= sn;i++){
		if(n % i === 0){
			return false;
		}
	}
	return true;
}

console.log("Primes.");
let c = 0;
for(let i = 2;i <= 1000000;i++){
	if(isPrime(i)){
		c++;
	}
}
console.log("Done.",c);